src/Orchard.Web/Modules/Orchard.MediaLibrary/Permissions.cs

```diff
@@ -4,13 +4,15 @@
 
 namespace Orchard.MediaLibrary {
     public class Permissions : IPermissionProvider {
-        public static readonly Permission ManageMediaContent = new Permission { Description = "Managing Media", Name = "ManageMediaContent" };
+        public static readonly Permission ManageMediaContent = new Permission { Description = "Manage Media", Name = "ManageMediaContent" };
+        public static readonly Permission ManageOwnMedia = new Permission { Description = "Manage Own Media", Name = "ManageOwnMedia", ImpliedBy = new[] { ManageMediaContent } };

```

Comment:
I think the other one is the issue, but don't want to change it or it would break.